iroha/src/torii.rs

```diff
@@ -108,6 +114,23 @@ async fn handle_request(state: State<ToriiState>, request: Request) -> Result<Re
                 Ok(INTERNAL_ERROR.to_vec())
             }
         },
+        BLOCKS_URI => match Message::try_from(request.payload().to_vec()) {
```

Comment:
Can we use fully qualified name here, it's difficult to understand what `Message` it is.